figures/label_sampling_experiment/plot_scripts/class_accuracy_figure_generator.py

```diff
@@ -0,0 +1,123 @@
+
+
+file_tree = OrderedDict({
+    "DaG":
+    {
+        "DaG": "../../../disease_gene/disease_associates_gene/label_sampling_experiment/results/DaG/results",
+        "CtD": "../../../disease_gene/disease_associates_gene/label_sampling_experiment/results/CtD/results",
+        "CbG": "../../../disease_gene/disease_associates_gene/label_sampling_experiment/results/CbG/results",
+        "GiG": "../../../disease_gene/disease_associates_gene/label_sampling_experiment/results/GiG/results",
+        "All": "../../../disease_gene/disease_associates_gene/label_sampling_experiment/results/all",
+    },
+    "CtD":
+    {
+        "DaG": "../../../compound_disease/compound_treats_disease/label_sampling_experiment/results/DaG/results",
+        "CtD": "../../../compound_disease/compound_treats_disease/label_sampling_experiment/results/CtD/results",
+        "CbG": "../../../compound_disease/compound_treats_disease/label_sampling_experiment/results/CbG/results",
+        "GiG": "../../../compound_disease/compound_treats_disease/label_sampling_experiment/results/GiG/results",
+        "All": "../../../compound_disease/compound_treats_disease/label_sampling_experiment/results/all",
+    },
+
+    "CbG":
+    {
+        "DaG": "../../../compound_gene/compound_binds_gene/label_sampling_experiment/results/DaG/results",
+        "CtD": "../../../compound_gene/compound_binds_gene/label_sampling_experiment/results/CtD/results",
+        "CbG": "../../../compound_gene/compound_binds_gene/label_sampling_experiment/results/CbG/results",
+        "GiG": "../../../compound_gene/compound_binds_gene/label_sampling_experiment/results/GiG/results",
+        "All": "../../../compound_gene/compound_binds_gene/label_sampling_experiment/results/all",
+    },
+    "GiG":
+    {
+        "DaG": "../../../gene_gene/gene_interacts_gene/label_sampling_experiment/results/DaG/results",
+        "CtD": "../../../gene_gene/gene_interacts_gene/label_sampling_experiment/results/CtD/results",
+        "CbG": "../../../gene_gene/gene_interacts_gene/label_sampling_experiment/results/CbG/results",
+        "GiG": "../../../gene_gene/gene_interacts_gene/label_sampling_experiment/results/GiG/results",
+        "All": "../../../gene_gene/gene_interacts_gene/label_sampling_experiment/results/all",
+    }
+})
+
+performance_data_tree = OrderedDict({
+    key: {
+        sub_key: get_dataframes(file_tree[key][sub_key], "*sampled_performance.tsv")
+        for sub_key in file_tree[key]
+    }
+    for key in file_tree
+})
+
+plt.rcParams.update({'font.size': 22})
+
+class_correct_data_tree = OrderedDict({
+    key: {
+        sub_key: get_dataframes(file_tree[key][sub_key], "*marginals.tsv")
+        for sub_key in file_tree[key]
+    }
+    for key in file_tree
+})
+
+
+plot_performance_graph(
+    metric="frac_correct", evaluation_set='dev',
+    title="Individual Class Performance (Dev Set)",
+    file_name="class_correct_dev_set.png", data=class_correct_data_tree
+)
+
+plot_performance_graph(
```

Comment:
So these .py files are not being called from your main nb?  In terms of organization i usually like to do that.  But I'm not seeing the overall big picture of this repo

Reply:
I like the plan to make these scripts actual notebooks instead. Given the current time constraint, I'll push this idea onto the immediate todo list pile.